sass/layout/_sidebar-left.scss

```diff
@@ -335,6 +335,61 @@
         }
     }
 
+    .SidebarCategory_newLabel {
+        font-size: 10px;
+        font-weight: 600;
+        line-height: 16px;
+        display: flex;
+        align-items: center;
+        letter-spacing: 0.01em;
+        text-transform: uppercase;
+        color: var(--mention-color);
+        border-radius: 4px;
+        background: var(--mention-bg);
+        width: 32px;
+        height: 16px;
+        padding: 0 4px;
+        margin-left: 4px;
+        margin-right: 16px;
+    }
+
+    .SidebarCategory_newDropBox {
+        border: 1px dashed var(--sidebar-text-30);
+        border-radius: 4px;
+        margin: 0 16px;
+        padding: 10px;
+        display: flex;
+        align-items: center;
+
+        > i {
+            font-size: 18px;
+            line-height: 16px;
+            color: var(--sidebar-text-60);
+        }
+
+        .SidebarCategory_newDropBox-label {
+            font-size: 12px;
+            line-height: 16px;
+            color: var(--sidebar-text-72);
+            margin-left: 8px;
+        }
+
+        &.animating {
+            transition-property: height, padding, color;
+            transition-duration: 0.15s; /* collapse animation speed */
+            transition-timing-function: ease-out;
+        }
+
+        &.collapsed {
+            height: 0 !important;
```

Comment:
I'm assuming the `!important` is absolutely necessary here? If so, I'm wondering if we should maybe have. a comment to explain why it is being used?

Reply:
Yeah I can add a comment.